onnx/common/tensor.h

```diff
@@ -75,6 +75,8 @@ struct Tensor final {
     name_ = std::string(other.name_.data(), other.name_.size());
     raw_data_ = std::string(other.raw_data_.data(), other.raw_data_.size());
   }
+  Tensor(Tensor&&) = default;
```

Comment:
Performance improvement: move constructor and move assignment operator are implicit deleted if copy constructor or copy assignment operator is explicitly declared, so move constructor should be defined explicitly to enable moving.